pkg/util/gitutil/git.go

```diff
@@ -31,6 +31,21 @@ import (
 	"gopkg.in/src-d/go-git.v4/storage/memory"
 )
 
+// Constants related to detecting the right type of source control provider for git.
+const (
+	defaultGitCloudRepositorySuffix = ".git"
+
+	// The host name for GitLab.
+	GitLabHostName = "gitlab.com"
+	// The host name for GitHub.
+	GitHubHostName = "github.com"
+)
+
+// The pre-compiled regex used to extract owner and repo name from an SSH git remote URL.
+// CAUTION! If you are renaming any of the group names in the regex (the ?P<group_name> part) to something else,
```

Comment:
I do appreciate the `CAUTION!` sentiment, as it would be helpful. But since we have some good tests for this package, it isn't really necessary. As if the group names were changed in the regex, the tests would immediately start failing (and be pretty clear what the culprit was). So I'd suggest removing that part of the comment.

Reply:
While the failing tests will indicate that something is wrong, I feel it is not immediately obvious that the problem might be with the group names. Could we still keep this comment? If you feel strongly about removing it, I shall remove it.

Reply:
Feel free to keep the comment. But "CAUTION!" might be a bit too strong of an indicator. Perhaps just something like "Note that the group names are referenced in the code below, so be sure to change them together." or something?

Reply:
Got it. Will do that.